examples/using-guidebot-flags.md

```diff
@@ -0,0 +1,106 @@
+# Using Switch Cases with Guidebot
+
+This page is an explanation of how to use switch cases with a feature in Guidebot, `message.flags`.
+
+## An Example
+
+Switch cases are a nice feature that can be used without the `message.flags` feature, and can be used like this. 
+
+```js
+// This will look for the first argument. If your command was 'hi', and you did 'hi send', it would send 'Hi!' to the channel.
+const trigger = args[0];
+
+switch (trigger) {  
```

Comment:
This is inconsistent with the code below. `switch(args[0])` would be better.

Reply:
Fixed that.